lib/msal-node/src/network/HttpClient.ts

```diff
@@ -23,21 +25,39 @@ export class HttpClient implements INetworkModule {
      */
     async sendGetRequestAsync<T>(
         url: string,
-        options?: NetworkRequestOptions
+        options?: NetworkRequestOptions,
+        cancellationToken?: number,
+        proxy?: string
```

Comment:
Can `proxy` and `cancellationToken` be added to `NetworkRequestOptions` instead? I would like us to get out of the habit of many optional parameters like this.